lib/chef/azure/commands/enable.rb

```diff
@@ -117,27 +120,44 @@ def configure_chef_only_once
         f.write(@validation_key)
       end
 
-      # Write client.rb
-      File.open("#{bootstrap_directory}/client.rb", "w") do |f|
-        f.write(override_clientrb_file(@client_rb))
-      end
-      # write the first_boot.json
-      File.open("#{bootstrap_directory}/first-boot.json", "w") do |f|
-        f.write(<<-RUNLIST
-{
-"run_list": [#{ @run_list.empty? ? "" : escape_runlist(@run_list)}]
-}
-RUNLIST
-)
+      if windows?

```

Comment:
Why is this conditional on Windows? Shouldn't we do the same thing on all platforms?